src/AppBundle/Controller/DeleteCounterController.php

```diff
@@ -0,0 +1,198 @@
+     */
+    public function indexAction(Request $request, $project = null)
+    {
+        // Get the query parameters.
+        $projectName = $project ?: $request->query->get('project');
+        $username = $request->query->get(
+            'username',
+            $request->query->get('user')
+        );
+
+        // If we've got a project and user, redirect to results.
+        if ($projectName != '' && $username != '') {
+            $routeParams = [ 'project' => $projectName, 'username' => $username ];
+            return $this->redirectToRoute(
+                'DeletionCounterResult',
+                $routeParams
+            );
+        }
+
+        // Instantiate the project if we can, or use the default.
+        $theProject = (!empty($projectName))
+            ? ProjectRepository::getProject($projectName, $this->container)
+            : ProjectRepository::getDefaultProject($this->container);
+
+        // Show the form.
+        return $this->render(
+            'deletionCounter/index.html.twig',
+            [
+            'xtPageTitle' => 'tool-dc',
+            'xtSubtitle' => 'tool-dc-desc',
+            'xtPage' => 'dc',
+            'project' => $theProject,
+            ]
+        );
+    }
+
+    /**
+     * Display the results
+     *
+     * @param string $project  The project domain name.
+     * @param string $username The username.
+     *
+     * @return Response
+     *
+     * @Route("/dc/{project}/{username}", name="DeletionCounterResult")
+     */
+    public function resultAction($project, $username)
+    {
+        /**
+         * Project information
+         *
+         * @var Project $project
+         */
+        $project = ProjectRepository::getProject($project, $this->container);
+        $projectRepo = $project->getRepository();
+
+        $user = UserRepository::getUser($username, $this->container);
+
+        $userID = $user->getId($project);
+
+        if (!$project->exists()) {
+            $this->addFlash('notice', ['invalid-project', $project]);
+            return $this->redirectToRoute('SimpleEditCounter');
+        }
+
+        $dbName = $project->getDatabaseName();
+
+        $loggingTable = $projectRepo->getTableName(
+            $dbName,
+            "logging",
+            "userindex"
+        );
+
+        /**
+         * Connection to the database
+         *
+         * @var Connection $conn
+         */
+        $conn = $this->get('doctrine')->getManager('replicas')->getConnection();
+
+        $types = $this->container->getParameter("deletion_counter");
+
+        if (!isset($types[$project->getDatabaseName()])) {
+            $this->addFlash('notice', [ 'no-result', $username ]);
+            return $this->redirectToRoute(
+                'DeletionCounter',
+                [
+                    'project' => $project->getDomain()
+                ]
+            );
+        }
+
+        $data = $types[$project->getDatabaseName()];
```

Comment:
You might reuse `$dbName` here instead of calling `$project->getDatabaseName()` again, but personally I think the config should go by domain to be consistent with semi_automated.yml and rfx_analysis.yml.